pyproject.toml

```diff
@@ -78,6 +78,9 @@ tag_name = "v{new_version}"
 [[tool.bumpversion.files]]
 filename = "pyproject.toml"
 
+[tool.ruff.lint]
+exclude = ["docs", "tests", '*.ipynb']
```

Comment:
im pro linting everything (mainly tests is the one im focused on here) since it cuts down on diffs

Reply:
just removing notebooks (i have many in my local repo that i don't checkin but tests fail locally when i don't exclude them)